iree/task/worker.c

```diff
@@ -263,10 +263,10 @@ static bool iree_task_worker_pump_once(
   // Since we can host work from multiple scopes and want to ensure an error
   // in one doesn't bring down the whole system we pretend we executed
   // something here by falling through.
-  IREE_ASSERT_TRUE(iree_status_is_ok(status));
   if (!iree_status_is_ok(status)) {
     iree_status_fprint(stderr, status);
   }
+  IREE_ASSERT_TRUE(iree_status_is_ok(status));
```

Comment:
also unrelated, but fine :P

Reply:
I ended up needing to debug through this and was finding that having the actual status printed _before_ the debugger took over was useful :P